examples/verify-contract/index.js

```diff
@@ -15,16 +15,17 @@ function printVerificationToConsole(verification) {
   } else {
     console.log('No verifications available for this contract');
   }
+  console.log();
```

Comment:
🤔

Reply:
So you get a newline at the end of each block!

Reply:
ah clever